### PR TITLE
Signup code review

### PR DESCRIPTION
**Pull request by ErkoRisthein**

### CODE REVIEW

File: src/main/java/ee/tuleva/onboarding/user/UserController.java [hunk 1]
```diff
@@ -31,32 +31,32 @@ public UserResponse me(@ApiIgnore @AuthenticationPrincipal AuthenticatedPerson a
...
-				.email(cmd.getEmail())
-				.personalCode(cmd.getPersonalCode())
-				.phoneNumber(cmd.getPhoneNumber())
-				.createdDate(Instant.now())
-				.updatedDate(Instant.now())
-				.active(false)
-				.build();
-
-		return userRepository.save(newUser);
+		User user = userRepository.findByPersonalCode(authenticatedPerson.getPersonalCode());
```
Comment (JordanValdma) [inline src/main/java/ee/tuleva/onboarding/user/UserController.java]: extract into a service?

File: src/main/java/ee/tuleva/onboarding/user/UserController.java [hunk 2]
```diff
@@ -31,32 +31,32 @@ public UserResponse me(@ApiIgnore @AuthenticationPrincipal AuthenticatedPerson a
...
+		user.setEmail(cmd.getEmail());
+		user.setPhoneNumber(cmd.getPhoneNumber());
+
+		try {
```
Comment (JordanValdma) [inline src/main/java/ee/tuleva/onboarding/user/UserController.java]: maybe not needed?

File: src/main/java/ee/tuleva/onboarding/user/response/UserResponse.java [hunk 3]
```diff
@@ -16,22 +17,34 @@
...
 
     public int getAge() {
-        return PersonalCode.getAge(this.getPersonalCode());
+        return PersonalCode.getAge(personalCode);
     }
 
     public static UserResponse fromAuthenticatedPerson(AuthenticatedPerson authenticatedPerson) {
-        return UserResponse.builder()
+        return builder()
                 .id(authenticatedPerson.getUser().getId())
```
Comment (JordanValdma) [inline src/main/java/ee/tuleva/onboarding/user/response/UserResponse.java]: should write TODO here to add phone number/email OR remove this method and replace usage with new method all together

Comment (ErkoRisthein) [inline src/main/java/ee/tuleva/onboarding/user/response/UserResponse.java]: 🍺

**Pull request merged**